src/Resource/Resource.php

```diff
@@ -101,6 +101,10 @@ public function getContextForProcessInSinglePlace()
      */
     private function isLocal()
     {
+        if(is_resource($this->original)){
```

Comment:
To keep in line with the library's coding style, it should be `if (is_resource($this->original)) {`